sdk/core/Azure.Core/src/ResponseClassifier.cs

```diff
@@ -28,6 +28,16 @@ public virtual bool IsRetriableException(Exception exception)
                    (exception is RequestFailedException requestFailed && requestFailed.Status == 0);
         }
 
+        /// <summary>
+        /// Specifies if the operation that caused the exception should be retried taking the <see cref="HttpMessage"/> into consideration.
+        /// </summary>
+        public virtual bool IsRetriableException(HttpMessage message, Exception exception)
```

Comment:
Hmm, so the other overload should never be used now?

Reply:
Not, it can still be used in situations where HttpMessage is not available. Retryable stream would keep using it.